### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by gergitokusse**
### 👇 👇 The following feature was added to the branch 👇 👇 
👉 New DB named `Vet_Clinic` were created
👉 New table named `animals` was created
👉 Four data items were inserted into the table
👉 SQL queries were added to the corresponding file in the template file.
👉 Query based on the requirement was done and the following images are the result of the query:-

image

image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -3,3 +3,4 @@ INSERT INTO animals VALUES(1, 'Agumon', '2020-02-3', 0, true, 10.23);
 INSERT INTO animals VALUES(2, 'Gabmon', '2018-11-15', 2, true, 8);
 INSERT INTO animals VALUES(3, 'pikachu', '2021-01-7', 1, false, 15.04);
 INSERT INTO animals VALUES(4, 'Deveimon', '2017-05-12', 5, true, 11);
+/* INSERT INTO animals VALUES(5, 'Charmander', '2020-02-08', 0, false, -11, ''),(6, 'Plantmon', '2021-11-15', 2, true, -5.7,''),(7, 'Squrtile', '1993-04-02', 3, false, -12.13, ''),(8, 'Angemon', '2005-05-12', 1, true, -45, ''),(9, 'Boarmon', '2005-06-07', 7, false, 20.4, ''),(10, 'Blessom', '1998-10-13', 3, true, 17, ''),(5, 'Ditto', '2022-05-14', 4, true, 22, ''); */
```
Comment (brainconnect93) [inline data.sql]: - [ ] Kindly remove all commented codes from your code base in order for your work to be free from being messy. this also gives a clean code base

File: schema.sql [hunk 2]
```diff
@@ -1,3 +1,4 @@
 /* Database schema to keep the structure of entire database. */
 CREATE DATABASE vet_clinic;
 CREATE TABLE animals(id INT NOT NULL, name VARCHAR(30) NOT NULL, date_of_brith DATE NOT NULL, escape_attempts INT NOT NULL, neutered BOOLEAN NOT NULL, weight_kg DEC(11,2) NOT NULL);
```
Comment (brainconnect93) [inline schema.sql]: - [ ] [Optional] I would kindly suggest you limit the `NOT NULL` parameter only to the `id` and also mark the `id` column as a primary key. i.e ``PRIMARY KEY (id)``

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @gergitokusse, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Animal table created with all necessary data✔️
- Screenshot of `queries` result added to the PR ✔️ 
- Good use of Git flow ✔️

### Changes Required

- [ ] Great effort working on your project, but there is still something you need to work on in order to get your project approved.

1. In your queries.sql file, the third query state that we should list the name of all animals that are neutered and have less than 3 escape attempts.
i.e 
```RUBY
SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';
```
but in your case, you checked for the escape_attempts to be greater than 3

i.e 
```RUBY
SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts > '3';
```

Kindly fix it and upload the correct query result screenshot on your PR.

2. Kindly work on the MIT license link as the link is currently broken and throwing error 404, also in the `show your support` heading in your readme there is no content there and it might look unprofessional when users visit your repo. Kindly work on it so as to have a professional readme file.

MIT license 👇 

mit



- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @gergitokusse , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Grat work for impelemtimg the changes as requested by the previous reviewer :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gergitokusse** [comment]
Thank you very much!!!

**Pull request merged**